### PR TITLE
MTM-48465 - documenation for tenant sso configuration access 

### PR DESCRIPTION
**Pull request by chbakar**
documented tenant SSO configuration access

### CODE REVIEW

File: content/users-guide/administration-bundle/changing-settings.md [hunk 1]
```diff
@@ -249,7 +249,20 @@ In order to use the single sign-on feature for {{< enterprise-tenant >}}s, the e
 
 To enable the feature, the administrator must configure a connection with the authorization server. This is done in the Administration application.
 
-Click the **Single sign-on** tab in the **Authentication** page.
+##### Configuration access
+
+SSO configurations can be configured to be exclusively accessible by the {{< management-tenant >}}, thus preventing other tenants from accessing the configurations.
+Users of such tenants will be unable to update the configuration. This removes the risk of an incorrectly configured SSO, which can prevent other users from login via SSO.
```
Comment (rssag) [inline content/users-guide/administration-bundle/changing-settings.md:255]:
```suggestion
Users of such tenants are unable to update the configuration. This removes the risk of an incorrectly configured SSO, which can prevent other users from logging in via SSO.
```

File: content/users-guide/administration-bundle/changing-settings.md [hunk 2]
```diff
@@ -249,7 +249,20 @@ In order to use the single sign-on feature for {{< enterprise-tenant >}}s, the e
...
+The {{< management-tenant >}} can grant or restrict access to SSO configurations for specific tenants, by assigning the following option to the tenants:
+
+- category: `sso`
+- key: `auth.sso.config.access.only.for.management`
+
+With this option enabled (set to `true`), when a tenant tries to access an SSO configuration which is only accessible by the {{< management-tenant >}}, the access for that particular tenant is denied.
+
+##### Configuration view
+
+Click the **Single sign-on** tab (this tab is only shown for tenants which have access to the SSO configuration) in the **Authentication** page.
```
Comment (rssag) [inline content/users-guide/administration-bundle/changing-settings.md:265]:
```suggestion
Click the **Single sign-on** tab in the **Authentication** page.
Note that the tab is only visible for tenants which have access to the SSO configuration. 
```

**Review by rssag — CHANGES_REQUESTED** [review]

**Review by rssag — APPROVED** [review]

**Comment by rssag** [comment]
which releases does this apply to?

**Comment by chbakar** [comment]
> which releases does this apply to?

10.16

**Review by Michal-Bak — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Review by Michal-Bak — APPROVED** [review]

File: content/users-guide/administration-bundle/changing-settings.md [hunk 3]
```diff
@@ -249,7 +249,16 @@ In order to use the single sign-on feature for {{< enterprise-tenant >}}s, the e
 
 To enable the feature, the administrator must configure a connection with the authorization server. This is done in the Administration application.
 
+##### Configuration access
+
+SSO configurations can be configured to be exclusively accessible by the {{< management-tenant >}}, thus preventing other tenants from accessing the configurations.
+Users of such tenants are unable to update the configuration. This removes the risk of an incorrectly configured SSO, which can prevent other users from logging in via SSO.
+The {{< management-tenant >}} can grant or restrict access to SSO configurations for specific tenants. For more information about configuration access, please refer to the [Login options API](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#operation/putAccessLoginOptionResource) in the {{< openapi >}}.
```
Comment (rssag) [inline content/users-guide/administration-bundle/changing-settings.md:256]:
```suggestion
The {{< management-tenant >}} can grant or restrict access to SSO configurations for specific tenants. For more information about configuration access, refer to the [Login options API](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#operation/putAccessLoginOptionResource) in the {{< openapi >}}.
```

**Review by rssag — APPROVED** [review]

**Pull request merged**